pkg/policy/k8s/service.go

```diff
@@ -57,7 +41,7 @@ func (p *policyWatcher) updateToServicesPolicies(svcID k8s.ServiceID, newSVC, ol
 	var errs []error
 
 	// Bail out early if updated service is not selectable
```

Comment:
Is the comment still accurate?

Reply:
Thanks for the reminder. I’ll delete this line.